src/actions/posts.js

```diff
@@ -360,7 +360,18 @@ export function getPosts(channelId, page = 0, perPage = Posts.POST_CHUNK_SIZE) {
             return null;
         }
 
-        dispatch(batchActions([
+        const actions = [];
+        if (page === 0 && Object.keys(posts.posts).length === Posts.POST_CHUNK_SIZE) {
```

Comment:
I don't think we need the length check do we? What happens in a newer channel with only a few posts?